samcli/commands/check/lib/ask_question.py

```diff
@@ -0,0 +1,30 @@
+"""
+A utility method used to ask the user a question. An int is returned only
+if it is within the specified range.
+"""
+
+import click
+
+
+def ask(question: str, min_val: int = 1, max_val: float = float("inf")) -> int:
+    """Prompts the user with a question for some input. A min and max value
+    can be passed and used.
+
+    Args:
```

Comment:
Please update documentation to numpy style; https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html